src/pages/en.md

```diff
@@ -316,9 +316,16 @@ For example, Rails has a [`rails-dev-box`](https://github.com/rails/rails-dev-bo
 
 [Docker](https://www.docker.com) is also a good way to make dependency management simpler.
 
-- [ ] Promptly update version of gem at RubyGems
+- [ ] Don't forget to update version of gem at RubyGems
```

Comment:
I think, we need more general title here.

Maybe, just "Release it (timely)!"?

Reply:
"Update your version on RubyGems in time" — my 5 cents